Lib/fontbakery/specifications/hhea.py

```diff
@@ -47,10 +47,15 @@ def com_google_fonts_check_073(ttFont):
   id = 'com.google.fonts/check/079',
   conditions = ['glyph_metrics_stats']
 )
-def com_google_fonts_check_079(ttFont):
+def com_google_fonts_check_079(ttFont, glyph_metrics_stats):
   """Monospace font has hhea.advanceWidthMax equal to each glyph's
   advanceWidth?"""
 
+  seems_monospaced = glyph_metrics_stats["seems_monospaced"]
+  if not seems_monospaced:
+    yield PASS, ("Font is not monospaced.")
```

Comment:
please yield a SKIP here, instead of PASS

* PASS means "this font is OK regarding this problem"
* SKIP means "this check does not apply to this specific font"